manifests/z/Zwylair/BetterDiscordAutoInstaller/1.3.0/Zwylair.BetterDiscordAutoInstaller.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Zwylair.BetterDiscordAutoInstaller
+PackageVersion: 1.3.0
+Installers:
+- Architecture: x64
+  InstallerType: portable

```

```suggestion
  InstallerType: zip
  NestedInstallerType: portable
  NestedInstallerFiles: 
    - RelativeFilePath: "BetterDiscordAutoInstaller v1.3.0\\main.exe"
      # PortableCommandAlias: 
```
I'm not sure that this will work, due to the spaces in the `RelativeFilePath`. I believe spaces might not be allowed there.